registry-analysis/Readme.md

```diff
@@ -22,11 +22,112 @@ Examples:
+- Entries *should* contain links to the specification of the profile itself and its corresponding vocabulary
+- Entries can be added, changed, or deleted
+  - Although deletion is possible, entries are intended to be stable
+- Requests are made via an email to the WG chair, will be decided on in a WG meeting
+- Decisions are made via consensus, requestor is notified accordingly
+
+#### Media Source Extensions Byte Stream Format Registry
+
+This [document](https://www.w3.org/TR/mse-byte-stream-format-registry/) defines
+a registry that maps between MIME-type/subtype pairs and byte stream format
+specifications.
+These are relevant for
+[`SourceBuffer` objects](https://www.w3.org/TR/media-source/#sourcebuffer) which
+are used for cosuming audio and video streams.
+
+The registry, its entries and the modification process are specified as follows:
+
+- Each entry consists of MIME type/subtype which is mapped to a publicly available specification for a byte stream format. Furthermore, it has to be specified if a flag for generating timestamps needs to be set with the byte stream format.
+- It is recommended that the specification should be available without fees.
+- All requirements mentioned in the [Media Source Extensions](https://www.w3.org/TR/media-source/#byte-stream-formats) specification must be fulfilled.
+- The possibility to remove entries that fail to satisfy any of the mandatory requirements is explicitly mentioned.
+- New entries are submitted to a public mailing list where they are supposed to be discussed and evaluated before adding them to the registry. The process of approving a new entry is (apparently?) not formally specified.
+
+#### DID Specification Registries
+
+This [document](https://www.w3.org/TR/did-spec-registries/) defines
+a registry for all known global parameters, properties, and values used by the
+Decentralized Identifier ecosystem.
+
+The registry, its entries and the modification process are specified as follows:
+
+- New entries have to be submitted via a modification request (as part of a PR on the registry's repository)
+- Entries must include a human-readable description and should have a clear and descriptive name
+- Entries must adhere to an IPR policy and do no harm to others (including legally and security-wise).
+- Entries must link to a specification that makes it possible to implement the addition
+- Entries must specify a JSON-LD context (with a set of more finegrained requirements)
+- Entries must not be removed, only deprecated
+<!-- TODO: Could maybe be expanded -->
+
+#### XPointer Scheme Name Registry Policy
+
+This [document](https://www.w3.org/2005/04/xpointer-policy) defines a policy
+that governs the registration of simple names for Scheme-Based XPointers.
+
+The registry, its entries and the modification process are specified as follows:
+
+- Entries are given out on a "first come, first serve" basis
```

Comment:
This is interesting for us and did not see this in other mechanisms. This can help with the case that two people try to register two entries (one each) for a certain protocol. E.g. what do we do if another person says "I don't like the CoAP Binding Template, here is mine which has other design decisions and thus not a direct improvement of the existing document"